assets/embauche.html

```diff
@@ -107,177 +107,179 @@
 	<details id="taxes" class="js-only">
 		<summary>Cotisations détaillées</summary>
 
-		<table>
-			<caption>Cotisations employeur</caption>
+		<div class="tables">
```

Comment:
Using this additional wrapper without documentation for working around a polyfill-induced bug is not maintainable.

The `class` is used nowhere but introduces possibilities for client styling conflicts, and nothing explicits its reason to be. The day Firefox implements `<details>` natively, no one will remember to remove this wrapper div.

What would you say about looking into a CSS-based solution? I guess the issue is that the polyfill [forces `display` to `block` on `details > *` when it's open](https://github.com/MattiSG/Element.details/blob/mattisg/__SRC/Element.details.js#L37). Maybe we can improve the polyfill by making it set `display` to `initial` or `unset`, or, as a worst case, override the inner styling with a `! important` adjustment in the stylesheet that explains it is here just for FF compat.

WDYT?